core/services/offchainreporting/OCRContract.go

```diff
@@ -0,0 +1,228 @@
+package offchainreporting
+
+import (
+	"context"
+	"math/big"
+	"strings"
+	"sync"
+	"time"
+
+	ethereum "github.com/ethereum/go-ethereum"
+	gethCommon "github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink/core/services/eth"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+	"github.com/smartcontractkit/chainlink/offchainreporting/confighelper"
+	"github.com/smartcontractkit/chainlink/offchainreporting/gethwrappers/offchainaggregator"
+	ocrtypes "github.com/smartcontractkit/chainlink/offchainreporting/types"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/pkg/errors"
+)
+
+var (
+	OCRContractConfigSet = getConfigSetHash()
+)
+
+type (
+	LogConfigSet struct {
+		eth.GethRawLog
+		ConfigCount  uint64
+		Oracles      []gethCommon.Address
+		Transmitters []gethCommon.Address
+		Threshold    uint8
```

Comment:
`EncodedConfigVersion uint64` is missing

Reply:
Not sure how it can be missing... this was copied directly from ocr prototype

Reply:
Hmm, actually this entire struct might be dead code? I don't see any references to it.